pkgs/development/libraries/easyloggingpp/default.nix

```diff
@@ -10,9 +13,22 @@ stdenv.mkDerivation rec {
   };
   nativeBuildInputs = [cmake];
   buildInputs = [gtest];
-  cmakeFlags = [ "-Dtest=ON" "-Dbuild_static_lib=ON"];
+  cmakeFlags = [ "-Dtest=ON" ];
   NIX_CFLAGS_COMPILE = "-std=c++11" +
     stdenv.lib.optionalString stdenv.isLinux " -pthread";
+  postInstall = ''
+    mkdir -p $out/src
+    cp ../src/easylogging++.cc $out/src
+    mkdir -p $out/lib/pkgconfig
+    cat << EOF > $out/lib/pkgconfig/easyloggingpp.pc
+    Name: easyloggingpp
+    Description: A C++ Logging Library
+    Version: ${version}
+    prefix=$out
+    includedir=\''${prefix}/include
+    Cflags: -I\''${includedir}
+    EOF
```

Comment:
I think it would be useful to provide this pkgconfig file also upstream.
Then users don't have to care about on which platform they compile against easylogging.